pkg/asset/installconfig/gcp/validation.go

```diff
@@ -466,34 +467,32 @@ func validateRegion(client API, ic *types.InstallConfig, fieldPath *field.Path)
 	return nil
 }
 
-// ValidateCredentialMode checks whether the credential mode is
-// compatible with the authentication mode.
-func ValidateCredentialMode(client API, ic *types.InstallConfig) error {
+// ValidateCredentialMode The presence of `authorized_user` in the credentials indicates that no service account
+// was used for authentication and requires Manual credential mode.
```

Comment:
Nit: 
// ValidateCredentialMode validates setting of Manual credential mode when `authorized_user` in the 
// credentials indicates that no service account is being used for authentication